tvrenamr/cli/core.py

```diff
@@ -49,8 +49,8 @@ def print_version(ctx, param, value):
 @click.option('--show', help="Set the show's name (will search for this name).")
 @click.option('--show-override', help="Override the show's name (only replaces the show's name in the final file)")   # noqa
 @click.option('--specials', help='Set the show\'s specials folder (defaults to "Season 0")')
-@click.option('--symlink', is_flag=True, help="Create symbolic links instead of moving the files. Requires '--rename-dir'.") # noqa
-@click.option('-t', '--the', is_flag=True, help="Set the position of 'The' in a show's name to the end of the show name")   # noqa
+@click.option('--symlink', is_flag=True, default=None, help="Create symbolic links instead of moving the files. Requires '--rename-dir'.") # noqa
+@click.option('-t', '--the', is_flag=True, default=None, help="Set the position of 'The' in a show's name to the end of the show name")   # noqa
```

Comment:
Because otherwise the default value from click will override the config, even if the user didn't call that option. Check my comment on the issue https://github.com/ghickman/tvrenamr/issues/87!